ckanext/budgetviz/timeseries_ipfs/theme/public/timeseries_ipfs_view.js

```diff
@@ -0,0 +1,138 @@
+//Prepare data for Timeseries
+/* function mungeData converts data returned from d3.csv to stream data.
+ Each stream would represent one timeseries of a category. In Total six stream would be present*/
+
+var mungeData = function(data) {
+    var parseDate = d3.time.format("%Y").parse;
+    var mndata = [];
+    var partaicular, values_arr, temp, key;
+    for (i = 0; i < data.length; i += 1) {
+        key = Object.keys(data[i]);
+        key.shift();
+        particular = Object.create(null);
+        particular.key = data[i].Particulars;
+        particular.type = data[i].Particulars.substring(data[i].Particulars.indexOf("(") + 1, data[i].Particulars.indexOf(")"));
+        values_arr = [];
+
+        for (j = 0; j < key.length; j += 1) {
+            temp = Object.create(null);
+            temp.x = parseDate(key[j].substring(0, 4));
+            temp.y = parseFloat(data[i][key[j]]);
+
+            values_arr.push(temp);
+
+        }
+        particular.values = values_arr;
+
+        mndata.push(particular);
+    }
+
+    return mndata;
+};
+//Find possible selection categories. 
+var getSelections = function(mndata) {
+    var types = new Set();
+    for (i = 0; i < mndata.length; i += 1) {
+
+        types.add(mndata[i].type);
+    }
+    selection = mndata[0].type;
+    return Array.from(types);
+};
+
+//Get Stream data for selected selection.
+var getStreamData = function(mndata, selection) {
```

Comment:
get is generally used as a slug for functions/methods instead of variables. Can we rename it?